### PR TITLE
Laundry list

### PR DESCRIPTION
**Pull request by Tomperez98**

### CODE REVIEW

File: src/resonate/shells/poller.py [hunk 1]
```diff
@@ -27,9 +27,7 @@ def __init__(
 
     def _run(self) -> None:
         while True:
-            poll_url = (
-                f"{self._url}:8002/{self._scheduler.logic_group}/{self._scheduler.pid}"
-            )
+            poll_url = f"{self._url}:8002/{self._scheduler.group}/{self._scheduler.pid}"
```
Comment (dfarr) [inline src/resonate/shells/poller.py:30]: Just want to make sure we don't forget to make the port configurable, I'm guessing this PR is for renames and we can get to this in the next one?

File: src/resonate/stores/remote.py [hunk 2]
```diff
@@ -13,23 +13,68 @@
...
-        encoder: IEncoder[str, str] | None = None,
+        request_timeout: int,
+        encoder: IEncoder[str, str],
     ) -> None:
         self.url = url
         self._request_timeout = request_timeout
-        self._encoder = encoder or Base64Encoder()
+        self._encoder = encoder
+
+    def create_callback(
```
Comment (dfarr) [inline src/resonate/stores/remote.py:34]: Can we name this just "create"?

**Review by dfarr — DISMISSED** [review]
Looks great 🎉

File: src/resonate/shells/poller.py (hunk 1 above)
Comment (Tomperez98) [reply src/resonate/shells/poller.py:30]: Yes! This is next in line, together with creating a top level `Resonate` class

**Pull request merged**